src/os/39cb0_len_100.c

```diff
@@ -7,7 +7,7 @@
 extern u64 nuScStack[NU_SC_STACK_SIZE / sizeof(u64)];
 
 // probably should be at 0x8009A630, unless D_8009A630 (what's technically used in boot_idle) isn't nuIdleFunc
-BSS void (*nuIdleFunc)(void);
+extern void (*nuIdleFunc)(void);
```

Comment:
This produces a warning because its alr declared

Reply:
we'll want to have this here anyway once BSS is sorted out, so I think the warning is justified